src/auth-service/bin/www

```diff
@@ -10,6 +10,26 @@ const middlewareConfig = require("../config/middleware");
 var debug = require("debug")("auth-service:server");
 var http = require("http");
 var isEmpty = require("is-empty");
+const log4js = require('log4js');
```

Comment:
this is where the configurations of the tool are picked from the config file.....all at App start